### PR TITLE
added the-infinite-black and renoise to nixpkgs

### PR DESCRIPTION
**Pull request by cgadski**
My first pull request, hopefully I didn't screw up too badly.

### CODE REVIEW

File: pkgs/applications/audio/renoise/default.nix [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+              url = "http://files.renoise.com/demo/Renoise_3_0_1_Demo_x86_64.tar.bz2";
+              sha256 = "1q7f94wz2dbz659kpp53a3n1qyndsk0pkb29lxdff4pc3ddqwykg";
+            }
+          else
+            requireFile {
+              url = "http://backstage.renoise.com/frontend/app/index.html#/login";
+              name = "rns_3_0_1_linux_x86_64.tar.gz";
+              sha256 = "0n23m0f3pkwgicgvvg4hr7wa1jdk0py91fndda5986ri0km4j6af";
+            }
+        else throw "haven't written this for systems besides 64-bit linux";
```
Comment (matthiasbeyer) [inline pkgs/applications/audio/renoise/default.nix]: I _guess_ you can remove the `if current system is x86_64` thing by providing `meta.platforms`, but I'm not sure whether this keeps a non-x86_64 user from installing this!

File: pkgs/applications/audio/renoise/default.nix [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+      unpackPhase = ''
+        tar -xf $src
+      '';
```
Comment (matthiasbeyer) [inline pkgs/applications/audio/renoise/default.nix]: `unpackPhase` should do this automatically, I think!

File: pkgs/games/the-infinite-black/default.nix [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      name = "the-infinite-black";
+
+      src = 
+        if stdenv.system == "x86_64-linux" 
+          then
+            fetchurl {
+              url = "https://files.spellbook.com/download/tib-unity-linux.zip";
+              sha256 = "0cahbbw5d4fg6wirsckcjfwbzgsfb7g1dxngw95ldcdakj7pj5la";
+            }
+          else throw "haven't written this for systems besides x86_64 linux";
```
Comment (matthiasbeyer) [inline pkgs/games/the-infinite-black/default.nix]: Same thing as for the other patch here... try to set `meta.platforms`!

File: pkgs/games/the-infinite-black/default.nix [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+      phases = "unpackPhase installPhase";
+
+      unpackPhase = ''
+        unzip $src
+      '';
```
Comment (matthiasbeyer) [inline pkgs/games/the-infinite-black/default.nix]: Should also be done by the `unpackPhase` automatically.

**Comment by matthiasbeyer** [comment]
Make sure you clean up your git history before this gets merged, please! :smile:

**Comment by MagneticDuck** [comment]
ugh, I can't use git, restarting this

**Pull request closed without merging**